plugins/ScheduledReports/ReportEmailGenerator/AttachedFileReportEmailGenerator.php

```diff
@@ -47,11 +46,11 @@ protected function configureEmail(Mail $mail, GeneratedReport $report)
         $message = $this->getMessageBody($report);
         $mail->setBodyHtml($message);
 
-        $mail->createAttachment(
+        $mail->addAttachment(
             $report->getContents(),
             $this->attachedFileMimeType,
-            Zend_Mime::DISPOSITION_INLINE,
-            Zend_Mime::ENCODING_BASE64,
+            'inline',
+            Mail::ENCODING_BASE64,
```

Comment:
btw what does the encoding mean here? Looking at the usages it doesn't look like the body is actually eg base64 encoded? eg in plugins/ScheduledReports/ReportEmailGenerator/AttachedFileReportEmailGenerator.php 

Maybe the Mail method can decide on the encoding? Not sure we want to use other encodings? (haven't looked)

Reply:
yes. will remove that. PHPMailer will automatically encode the data based on the given encoding, so should be fine to always use base64